packages/admin-stories/.storybook/preview.tsx

```diff
@@ -8,9 +8,9 @@ import { addDecorator, addParameters } from "@storybook/react";
 import * as React from "react";
 import { IntlProvider } from "react-intl";
 import { MainContent, MuiThemeProvider } from "@comet/admin";
-import { Theme, createMuiTheme } from "@material-ui/core";
-import { getThemeOptions } from "@comet/admin-theme";
-import styled, { createGlobalStyle } from "styled-components";
+import { createMuiTheme } from "@material-ui/core";
+import { createGlobalStyle } from "styled-components";
+import { createCometTheme } from "@comet/admin-theme/lib/createCometTheme";
```

Comment:
Is that on purpose, importing from @comet/admin-theme/**lib** ?

Reply:
Ah, no 👍🏻